pymc/model.py

```diff
@@ -425,6 +426,16 @@ def profile(self):
         return self._aesara_function.profile
 
 
+class PointFunc:
```

Comment:
We can just put it in utils or aesaraf instead of moving it to the top of the module. Suggesting this because I was planning to add a dispatch for aesara dprint method as well...